### PR TITLE
Fix the skip message since it was the exact opposite of the tested co…

### PR DESCRIPTION
**Pull request by jcsirot**
…ndition

Signed-off-by: Jean-Christophe Sirot <jean-christophe.sirot@docker.com>

**What this PR does / why we need it**:
The displayed message was incorrect

**Release note**:

```
NONE
```

### CODE REVIEW

File: test/e2e/framework/util.go [hunk 1]
```diff
@@ -359,7 +359,7 @@ func SkipIfMultizone(c clientset.Interface) {
 		Skipf("Error listing cluster zones")
 	}
 	if zones.Len() > 1 {
-		Skipf("Requires more than one zone")
```
Comment (dims) [inline test/e2e/framework/util.go]: should it be `at most`? given the if condition

Comment (jcsirot) [reply test/e2e/framework/util.go]: Indeed, in the case there is no zone at all (i.e. no cloud provider integration I think) `zones` may be empty. I'm going to fix that.

**Review by yastij — APPROVED** [review]
/lgtm

thanks @jcsirot

**Pull request merged**